### PR TITLE
Adds a Github Action workflow

### PR DESCRIPTION
**Pull request by PiotrJander**

### CODE REVIEW

File: .github/workflows/neptune-contrib.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: neptune-contrib
+on: [push]
```
Comment (jakubczakon) [inline .github/workflows/neptune-contrib.yml:2]: I think this should on push to master only. 
Is it now? Seems like it is any push.

File: .github/workflows/neptune-contrib.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      - name: Install dependencies
+        run: |
+          pip install -r test_requirements.txt
+          pip install tox-travis pylintfileheader
+      - name: Build and test
+        run : |
+          # python -m pylint -j 0 -f parseable neptunecontrib tests
+          tox
+      - name: Publish package
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```
Comment (jakubczakon) [inline .github/workflows/neptune-contrib.yml:24]: Does this make it publish only if push was tagged?

**Review by kamil-kaczmarek — COMMENTED** [review]
I think that Contrib should be tested on Win/MacOS/Linux, similarly to neptune-examples.

File: .github/workflows/neptune-contrib.yml (hunk 2 above)
Comment (PiotrJander) [reply .github/workflows/neptune-contrib.yml:24]: Yup

**Comment by jakubczakon** [comment]
@kamil-kaczmarek right now there are not a lot of tests here so I am not sure if it matters a lot.

**Review by kamil-kaczmarek — APPROVED** [review]

**Review by jakubczakon — APPROVED** [review]

**Pull request merged**